src/CoWSwapEthFlow.sol

```diff
@@ -120,4 +125,26 @@ contract CoWSwapEthFlow is CoWSwapOnchainOrders, ICoWSwapEthFlow {
             revert EthTransferFailed();
         }
     }
+
+    /// @inheritdoc ICoWSwapEthFlow
+    function isValidSignature(bytes32 orderHash, bytes memory)
+        external
+        view
+        override(EIP1271Verifier, ICoWSwapEthFlow)
+        returns (bytes4)
+    {
+        // Note: the signature parameter is ignored since all information needed to verify the validity of the order is
```

Comment:
Maybe you should explicitly:
```solidity
require(signature.length == 0);
```

It has slightly different semantics ("only the empty signature is valid" vs "all signatures are valid"). Not sure if there is an attack surface for allowing multiple signatures to be valid.

Reply:
Overall I don't see an issue with allowing arbitrarily signatures to be valid, so I'd be more in favor of keeping it as it is to save a little bit of gas.

Reply:
A reason to restrict the signature lenght to 0 is that it would catch issue with signature encoding in the driver or in somebody who builds a solver at a lower level. Do you think this would be worth the little extra gas required?

Reply:
I think its fine - just thought I would bring it up.